src/gwt/src/org/rstudio/studio/client/workbench/codesearch/ui/CodeSearchDialog.java

```diff
@@ -46,13 +49,17 @@ public CodeSearchDialog(Provider<CodeSearch> pCodeSearch)
    
    @Override
    protected Widget createMainWidget()
-   {     
+   {  
       VerticalPanel mainPanel = new VerticalPanel();
       mainPanel.addStyleName(
          CodeSearchResources.INSTANCE.styles().codeSearchDialogMainWidget());
       codeSearch_ = pCodeSearch_.get();
       codeSearch_.setObserver(this);
       mainPanel.add(codeSearch_.getSearchWidget());
+
+      HelpLink help = new HelpLink(constants_.usingFuzzyFinder(), "navigating_code");
```

Comment:
Should this wait until https://github.com/rstudio/rstudio-web/pull/33 is merged ? and keep the actual link in the meantime ?

Reply:
Just merged the redirect now, thanks!